### DISCUSSION TITLE
Phantom Event

### DISCUSSION THREAD

**Discussion by mt-subnet** [kewisch/gdata-provider#571] [Support]
Receiving reminders for Deleted Event which can now not be deleted. 
Error Log entry:-
Request DELETE https://www.googleapis.com/calendar/v3/calendars/emailaddress%40usa/events/3aku8vthjkm5ric7u8ds9b65ld responded with HTTP 404
This seems a Google calendar problem but I have no idea how to tackle it.
Anyone?

**Comment by kewisch** [comment]
This seems to be related to https://github.com/kewisch/gdata-provider/issues/581. Can you add your findings there if they are not already covered?
